src/gatsby-theme-carbon/components/FeedbackDialog/FeedbackDialog.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import ThemeFeedbackDialog from 'gatsby-theme-carbon/src/components/FeedbackDialog/FeedbackDialog';
+import * as axios from 'axios';
+import { ToastNotification } from 'carbon-components-react';
+
+const REQUEST_KEY = 'b64cb88a-fab4-4e4f-8ab3-07574522a125';
+const SURVEY_ID = '5806027';
+
+const successMessage = {
+  title: 'Feedback sent',
+  subtitle: 'Thank you for submiting your feedback.',
+  kind: 'success',
+};
+const failMesage = {
+  title: 'Feedback not sent',
+  subtitle: 'Please contact the website administrator.',
+  kind: 'error',
+};
+
+let timeoutInstance;
+
+const FeedbackDialog = ({ props }) => {
+  const [notification, setNotification] = useState({});
+
+  const showNotification = (notificationOpt) => {
+    clearTimeout(timeoutInstance);
+    setNotification(notificationOpt);
+    timeoutInstance = setTimeout(() => {
+      setNotification({});
+    }, 4500);
+  };
+
+  const onSubmit = (feedbackData) => {
+    const config = {
+      method: 'post',
+      params: {
+        ...feedbackData,
+        surveyId: SURVEY_ID,
+      },
+      url: 'https://b3fa4946.us-south.apigw.appdomain.cloud/feedback/submit',
```

Comment:
Not sure this should be public either

Reply:
Sorry @vpicone . Wasn't notified about this :( Yes, it should be public. This is not the gizmo API KEY. This key is used for the ibm cloud to control the number of requests. Also the feedback can be submitted only from ibm domains. 

That url is a public one.

Reply:
@vpicone ☝️

Reply:
@vpicone @sstrubberg  I'm not sure what do I need to anymore...haven't heard of making a request without using an url 😅

Reply:
Also, the url is no threat from a security concern because it just point to a serverless function. The logic behind that endpoint is important.